### ISSUE TITLE
Is changing case of common initialisms a bad design ?

### ISSUE THREAD

**Issue by flintforge** [99designs/gqlgen#1700]
The list could go on forever, and it's not the job of the generator to stumble upon specific cases.
This is tied to a semantic context: for instance fields named **cpu** or **ip** may not be expected to be turned uppercase.
If I may, I would advice to let users provide themselves any list of strings they judge necessary to go through this transformation. Even the id case (be it Freudian or not, cf [templates:449](https://github.com/99designs/gqlgen/blob/28caa6ce504b4ed206407dcaea8f8418feb91888/codegen/templates/templates.go#L449)) is dubious IMHO.

**Comment by JonasDoe** [comment]
I agree on that, but I think I've got that covered in a - rather dusty - issue: https://github.com/99designs/gqlgen/issues/1383
